pkgs/applications/misc/cobang/default.nix

```diff
@@ -1,72 +1,72 @@
 { lib
-, atk
 , buildPythonApplication
 , fetchFromGitHub
-, gdk-pixbuf
+, brotlicffi
+, gst-python
+, kiss-headers
+, logbook
+, pillow
+, pygobject3, python-zbar
+, requests
+, single-version
 , gobject-introspection
 , gst-plugins-good
-, gst-python
 , gtk3
-, kiss-headers
 , libhandy
 , librsvg
-, logbook
 , networkmanager
-, pango
-, pillow
-, poetry-core
-, pygobject3
-, pytestCheckHook
+, setuptools
 , python
-, python-zbar
-, pythonRelaxDepsHook
-, requests
-, single-version
+, pytestCheckHook
 , wrapGAppsHook
 }:
 
 buildPythonApplication rec {
   pname = "cobang";
-  version = "0.10.1";
-  format = "pyproject";
+  version = "0.10.5";
+  pyproject = true;
 
   src = fetchFromGitHub {
     owner = "hongquan";
     repo = "CoBang";
     rev = "refs/tags/v${version}";
-    hash = "sha256-yNDnBTBmwcP3g51UkkLNyF4eHYjblwxPxS2lMwbFKUM=";
+    hash = "sha256-CfT/farNOJiWIioFBPx2q7bAFAE4khcojdZ7AsYaU6o=";
   };
 
-  pythonRelaxDeps = [
-    "logbook"
-    "Pillow"
-  ];
+  postPatch = ''
+    # Fixes "Multiple top-level packages discovered in a flat-layout"
+    sed -i '$ a\' pyproject.toml
```

Comment:
```suggestion
```
I don't think that makes a difference and less sed lines are probably easier to manage here